src/main/java/ch/njol/skript/classes/data/DefaultFunctions.java

```diff
@@ -56,6 +56,21 @@ private static String str(double n) {
 		Parameter<?>[] numbersParam = new Parameter[] {new Parameter<>("ns", DefaultClasses.NUMBER, false, null)};
 		
 		// basic math functions
+
+		Functions.registerFunction(new SimpleJavaFunction<Number>("average", new Parameter[] {new Parameter<>("list", DefaultClasses.NUMBER, false, null)}, DefaultClasses.NUMBER, true) {
```

Comment:
```suggestion
		Functions.registerFunction(new SimpleJavaFunction<Number>("average", numbersParam, DefaultClasses.NUMBER, true) {
```
There's already a static field for this type of parameter